pkg/minikube/cni/cilium.go

```diff
@@ -175,7 +175,7 @@ data:
   hubble-tls-key-file: /var/lib/cilium/tls/hubble/server.key
   hubble-tls-client-ca-files: /var/lib/cilium/tls/hubble/client-ca.crt
   ipam: "cluster-pool"
-  cluster-pool-ipv4-cidr: "10.0.0.0/8"
+  cluster-pool-ipv4-cidr: "10.244.0.0/16"
```

Comment:
thank you for fixing this
we should make "tempaltized" for example
```
"{{ .PodSubnet }}"
```
so if user passes extra args we should use that simmilar to here
https://github.com/kubernetes/minikube/blob/db4f0dc414087c1b646b95699280bbab9b61ed0c/pkg/minikube/bootstrapper/bsutil/ktmpl/v1beta2.go#L74

Reply:
Added the template for extra args .